esmvaltool/utils/cmorizers/obs/cmorize_obs_UWisc.ncl

```diff
@@ -31,104 +35,97 @@ begin
   ; Tier
   TIER = 3
 
-  ; Field
-  FIELD = "T2Ms"
-
   ; Period
   YEAR1 = 1988
   YEAR2 = 2007
 
-  ; MIP
-  MIP = "Amon"
-
   ; Selected variable (standard name)
-  VARS = (/"clwvi", "lwpStderr"/)
+  VAR = (/"clwvi", "lwpStderr", "clivi"/)
```

Comment:
If it works in v2 (haven't tried yet), we should call the output variable "lwp" because that's what it actually is. We can then also get rid of the dummy variable "clivi".

Reply:
Good point. I will try this and test the recipe.

Reply:
Done.